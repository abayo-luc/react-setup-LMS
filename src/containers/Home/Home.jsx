import React, { Component } from 'react';
import NavBar from '../../components/NavBar/NavBar';
import ArticleCard from '../../components/Card/ArticleCard';
import './style.scss';
import Footer from '../../components/Footer/Footer';

export default class Home extends Component {
  state = {};

  render() {
    return (
      <div id="home">
        <NavBar />
        <div className="container">
          <div className="row justify-content-md-center">
            <div className="col col-md-10">
              <ArticleCard />
            </div>
            <div className="col col-md-2">
              <h2>Hello world</h2>
            </div>
          </div>
        </div>
        <Footer />
      </div>
    );
  }
}
