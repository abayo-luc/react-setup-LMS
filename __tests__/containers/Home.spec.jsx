import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { Home } from '../../src/containers/Home/Home';

const props = {
  articles: {},
  fetchArticles: jest.fn(),
};
const warper = shallow(<Home {...props} />);
describe('Home container', () => {
  it('should match the snapshot', () => {
    expect(toJson(warper)).toMatchSnapshot();
  });
});
