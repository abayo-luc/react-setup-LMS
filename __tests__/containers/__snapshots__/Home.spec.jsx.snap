// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home container should match the snapshot 1`] = `
<div
  id="home"
>
  <NavBar />
  <div
    className="container"
  >
    <div
      className="row justify-content-md-center"
    >
      <div
        className="col col-md-10"
      >
        <ArticleCard />
      </div>
      <div
        className="col col-md-2"
      >
        <h2>
          Hello world
        </h2>
      </div>
    </div>
  </div>
  <Footer />
</div>
`;
